### PR TITLE
doc: Added CPD documentation.

### PR DESCRIPTION
**Pull request by ashcrow**
Import of the CPD process using reST and sphinx instead of the wiki.

### CODE REVIEW

File: doc/cpd/1.rst [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+User Story             Name          User story that would be used to implement the change.
+Acceptance Criteria    Name          Criteria that must be met for the change to be considered complete.
+References             Name          Any helpful external links.
+====================== ============= =============
+
+CPD Status
+~~~~~~~~~~
+The status of a CPD will be changed to Accepted if and when 75% or more of the
+active core development team gives the CPD a ``:+1:``. It is the job of the current
+development lead (or a development lead) to update the CPD to Accepted status
```
Comment (mbarnes) [inline doc/cpd/1.rst]: Is the parenthesized part meant to say something else?  Having "development lead" mentioned twice is confusing; I don't get the distinction.

**Review by mbarnes — COMMENTED** [review]
Just a question; not sure if it's an old typo.  Otherwise looks good.

File: doc/cpd/1.rst (hunk 1 above)
Comment (ashcrow) [reply doc/cpd/1.rst]: It was meant to specify at least one development lead if there was more than on. How about:

> It is the job of at least one of the development leads to update...

Comment (mbarnes) [reply doc/cpd/1.rst]: Yeah, that's clearer.

**Comment by ashcrow** [comment]
:arrow_up:

**Review by mbarnes — APPROVED** [review]

**Comment by ashcrow** [comment]
[[multi-host-mgr-2] [spike] Design host abstraction](https://trello.com/c/TvgomncK/579-multi-host-mgr-2-spike-design-host-abstraction)

**Pull request merged**